### ISSUE TITLE
Highlights for summary spans with 'unreadable' text only appear for a flash

### ISSUE THREAD

**Issue by simonvarey** [fizzstudio/ParaCharts#1107]
**Describe the bug**
When a span has text that isn't read by the screen reader, such as an empty span, whitespace, or just punctuation, then a ParaAction highlight on that span will only appear for a fraction of a second.

**To Reproduce**
1. Add a span to a summary with 'unreadable' text and a highlight ParaAction
2. Run narrative highlights
3. The highlight is only visible for a fraction of a second

**Expected behavior**
I **think** what should happen is that each span should have a minimum time, so that any highlights are noticeable.
